src/NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp

```diff
@@ -110,6 +110,42 @@ auto logical_partial_derivatives(const Variables<VariableTags>& u,
     -> std::array<Variables<DerivativeTags>, Dim>;
 // @}
 
+// @{
+/*!
+ * \ingroup NumericalAlgorithmsGroup
+ * \brief Computes the logical partial derivative of a tensor
+ *
```

Comment:
Perhaps explain the index notation reasoning for the resulting type

Reply:
I don't think we do this for any other of our derivative operations and it hasn't been questioned. I'm not sure why this one should be special. I'll add a note that the spatial derivative index is added to the front, but I'm not sure how much justification for that can be made or is necessary.

Reply:
All I'm looking for is a small note `The return type is a tensor with the new down-index before all other indices for \f$ \partial_a T_{b...}^{c...}\f$` or rephrased as you like.

Reply:
How about:
```
 * \brief Computes the logical partial derivative of a tensor, prepending the
 * spatial derivative index, e.g. for \f$\partial_i T_{a}{}^{b}\f$ the C++ call
 * is `get(i, a, b)`.
 * ```